lib/God/ActionMethods.js

```diff
@@ -801,6 +801,22 @@ module.exports = function(God) {
 
         var proc_env = God.clusters_db[id].pm2_env;
 
+        var action_exist = false;
+        proc_env.axm_actions.forEach(function(action) {
```

Comment:
Could be refactored to a simpler code : 

```javascript
const isActionAvailable = proc_env.axm_actions.find(action => action.action_name === cmd.msg) !== undefined
```

There is `console.log` call also

Reply:
done